docs/docs/dev_docs/getting_started/updating.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Updating
+---
+
+There are 4 components whose versions need to be kept compatible:
+1. Aztec Sandbox,
+2. Aztec CLI,
+3. Noir compiler `nargo`,
+4. Noir framework for Aztec contracts `aztec.nr`.
+
+## Updating Aztec Sandbox
+To update the sandbox to the latest version, simply run the curl command we used for installation again:
+```shell
+/bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+```
+
+It will download and start the latest version of sandbox.
+
+If you would like to use a fixed version of the sandbox, you can export the `SANDBOX_VERSION` environmental variable.
+If you are unsure what version to use go to [aztec-packages repository](https://github.com/AztecProtocol/aztec-packages/releases) and choose the `aztec-packages` release based on the changelog.
+
+Then set the `SANDBOX_VERSION` environmental variable to the version you want to use. E.g.:
+```shell
+export SANDBOX_VERSION=0.7.7
+```
+
+Now when you run the curl command it will use the version you specified.
+To verify that it's the case check the console output of the curl command.
+You should see the following line:
+```
+Aztec Sandbox 0.7.7 is now ready for use!
+```
+
+Alternatively you can open a new terminal and use aztec-cli to get the version.
+
+#include_code node-info yarn-project/end-to-end/src/cli_docs_sandbox.test.ts bash
```

Comment:
I'd suggest starting the sandbox, since it reports the version on the first line, and it doesn't depend on another component.

```
sandbox Setting up Aztec Sandbox v0.1.0, please stand by...
```

Reply:
I have it there as the main approach (see the lines above the sentence starting with "Alternatively"). Or do you think that having there both approaches makes it more cluttered?

Reply:
> Aztec Sandbox 0.7.7 is now ready for use!

Isn't this the output of the curl command after installation? I was referring to the output of starting the standbox. But it's true that most people will be running the sandbox by running the curl, so if it's displayed every time, there's no point in adding a mention to the "Setting up..." message. Feel free to disregard!

Reply:
Just tried starting sandbox without by running yarn start in yarn-project/aztec-sandbox and it gets shown as well so it's not specific to initial setup.